static/js/around-map.js

```diff
@@ -6,39 +6,40 @@ const map = new mapboxgl.Map({
   zoom: 2
 })
 
-// users interesting points
-const pois = new Worker('/static/js/webworker-around.js');
-
-pois.onmessage = informations => {
-  const [name, wifi, power, comment, lng, lat] = informations.data
+map.addControl(new mapboxgl.NavigationControl())
+map.addControl(new mapboxgl.GeolocateControl())
 
-  const el = document.createElement('div')
-  el.classList.add('marker')
+// users interesting points
+const worker = new Worker('/static/js/webworker-around.js')
```

Comment:
right. I wonder if you could not use a relative import here?
`const worker = new Worker('./webworker-around.js')`